testcases/cve/meltdown.c

```diff
@@ -281,7 +281,7 @@ find_kernel_symbol(const char *name)
 
 	if (uname(&utsname) < 0)
 		tst_brk(TBROK | TERRNO, "uname");
-	sprintf(systemmap, "/boot/System.map-%s", utsname.release);
+	snprintf(systemmap, sizeof(systemmap), "/boot/System.map-%s", utsname.release);
```

Comment:
`systemmap` is supposed to be null terminated. So you must reserve one byte for null and set it to zero.

Reply:
Actually as far as I can tell `snprintf()` always null terminates the output buffer, unless of course the buffer size passed is 0. So this patch looks good to me.

Reply:
@KunWuChan the patch is fine, but misses the signed-off-my line. See: https://www.kernel.org/doc/html/latest/process/submitting-patches.html?highlight=signed%20off#developer-s-certificate-of-origin-1-1

Reply:
thanks, I've update the comment.